packages/react/src/hooks/contracts/useContractReads.test.ts

```diff
@@ -6,24 +7,24 @@ import {
   renderHook,
   wagmigotchiContractConfig,
 } from '../../../test'
-import { UseContractReadsConfig, useContractReads } from './useContractReads'
+import { useContractReads } from './useContractReads'
 
-const contracts: UseContractReadsConfig['contracts'] = [
+const contracts = [
   {
     ...wagmigotchiContractConfig,
     functionName: 'love',
-    args: '0x27a69ffba1e939ddcfecc8c7e0f967b872bac65c',
+    args: ['0x27a69ffba1e939ddcfecc8c7e0f967b872bac65c'],
   },
   {
     ...wagmigotchiContractConfig,
     functionName: 'love',
-    args: '0xa5cc3c03994DB5b0d9A5eEdD10CabaB0813678AC',
+    args: ['0xa5cc3c03994DB5b0d9A5eEdD10CabaB0813678AC'],
   },
-  { ...wagmigotchiContractConfig, functionName: 'getAlive' },
+  { ...wagmigotchiContractConfig, functionName: 'getAlive', args: [] },
```

Comment:
From internal convo: do we need to pass an empty `args` here? Type error seems to disappear if we add `'getAlive' as const`.